PRS7001-8000/PRS7853Philoxen.xml

```diff
@@ -44,14 +49,24 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sex="1">
-               <persName>
-                  
-                  Philoxenus of Mabbog
-               </persName>
-               <occupation> church father </occupation>
+            <person sex="1" sameAs="wd:Q973954">
+               <persName xml:lang="gez" xml:id="n1">ፍልክሲኖስ፡ ዘሀገረ፡ ምንብግ፡</persName>
+               <persName xml:lang="gez" type="normalized" corresp="#n1">Fǝlǝksinos za-hagara Mǝnbǝg</persName>
+               <persName xml:lang="en" type="main" corresp="#n1">Philoxenus of Mabbog</persName>
+               <persName xml:lang="syr" corresp="#n1">ܦܝܠܠܘܟܣܝܢܘܣ ܕܡܒܘܓ</persName>
+               <faith type="Syriac"/>
+               <occupation type="ecclesiastic">writer</occupation>
+               <death>523</death>
+               <floruit> He was one of the most notable Syriac prose writers and a vehement champion of Miaphysitism. In
+               Ethiopia he is known above all as an author of a monastic treaty <ref type="work" corresp="LIT1399Filkes">Filkǝsyos, which
```

Comment:
I think that <ref type="work" corresp="LIT1399Filkes"> could be either an empty element or closed after  Filkǝsyos